.circleci/README.md

```diff
@@ -0,0 +1,11 @@
+##### Required evn variables
+
+* `CONTAINER_DEST_REPO_DIR` - directory in container where dest repo is initialized within container
+* `DEST_REPO` - destination repository for rendered pages
+* `GITHUB_EMAIL`
+* `GITHUB_TOKEN`
+* `CIRCLE_USERNAME`
```

Comment:
Is `CIRCLE_USERNAME` the same as the built in? https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables